tensorflow/core/kernels/BUILD

```diff
@@ -3883,7 +3883,8 @@ tf_kernel_library(
 tf_kernel_library(
     name = "summary_op",
     prefix = "summary_op",
-    deps = LOGGING_DEPS,
+    deps = LOGGING_DEPS + ["//tensorflow/c/kernels:merge_summary_op_lib", 
```

Comment:
This is a kernel library, so lets just add the kernel here. Can you remove the op dep above?
Add the `merge_summary_op_lib` dependency here instead:
https://cs.opensource.google/tensorflow/tensorflow/+/master:tensorflow/core/BUILD;l=675